JPS_Ontology/ontology/ontosensorloggermobileapp/OntoSensorLoggerMobileApp.csv

```diff
@@ -0,0 +1,53 @@
+OntoSensorLoggerMobileApp,TBox,,http://www.w3.org/2002/07/owl#imports,,,,,,
+http://www.ontology-of-units-of-measure.org/resource/om-2/Measure,Class,,,,,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,Measure
+http://www.ontology-of-units-of-measure.org/resource/om-2/Quantity,Class,,,,,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,Quantity
+http://www.ontology-of-units-of-measure.org/resource/om-2/Unit,Class,,,,,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,Unit
+http://www.ontology-of-units-of-measure.org/resource/om-2/SoundPressureLevelUnit,Class,,,,,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,SoundPressureLevelUnit
+http://www.opengis.net/ont/sf#Geometry,Class,,,,,,,http://www.opengis.net/ont/sf#,Geometry
+https://www.theworldavatar.com/kg/ontotimeseries/TimeSeries,Class,,,,,,,https://www.theworldavatar.com/kg/ontotimeseries/TimeSeries,Time Series
+https://www.theworldavatar.com/kg/ontodevice/RelativeBrightness,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Ratio,IS-A,,,,,https://www.theworldavatar.com/kg/ontodevice/,RelativeBrightness
+http://www.ontology-of-units-of-measure.org/resource/om-2/Ratio,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Quantity,IS-A,,,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,Ratio
+http://www.opengis.net/ont/sf#Point,Class,http://www.opengis.net/ont/sf#Geometry,IS-A,,,,,http://www.opengis.net/ont/sf#,Point
+http://www.ontology-of-units-of-measure.org/resource/om-2/Speed,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Quantity,IS-A,,,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,Speed
+http://www.ontology-of-units-of-measure.org/resource/om-2/MagneticFluxDensity,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Quantity,IS-A,,,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,MagneticFluxDensity
+http://www.ontology-of-units-of-measure.org/resource/om-2/Acceleration,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Quantity,IS-A,,,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,Acceleration
+http://www.ontology-of-units-of-measure.org/resource/om-2/Illuminance,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Quantity,IS-A,,,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,Illuminance
+http://www.ontology-of-units-of-measure.org/resource/om-2/SoundPressureLevel,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Quantity,IS-A,,,,,http://www.ontology-of-units-of-measure.org/resource/om-2/,SoundPressureLevel
+MagneticFluxDensity_x,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/MagneticFluxDensity,IS-A,,,,Measures the direction of magnetic north with respect to the device in the x direction,http://www.theworldavatar.com/kg/sensorloggerapp,MagneticFluxDensity_x
```

Comment:
No longer sure this is a good approach. Probably better to define a MagneticFluxDensityVector, with hasXComponent etc., each of which is an om:MagneticFluxDensity...
Same applies to Gravity and Acceleration below.